cogs/mafia/mafia.py

```diff
@@ -66,10 +66,10 @@ async def leave(self, ctx: commands.Context):
 
     @commands.command()
     @game_only()
-    async def playerlist(self, ctx: commands.Context):
+    async def list_players_life_status(self, ctx: commands.Context):
```

Comment:
I think the change here is unnecessary. This method is used by `discord.py` to create a command, it's not intended to be used directly. `playerlist` sounds perfect for this command's name.

However, if we do end up changing the method name, then a different name should be used for the command using the `name` kwarg. `list_players_life_status` is too verbose.